unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetSearchValueRendererTest.ts

```diff
@@ -36,6 +37,7 @@ export function DynamicFacetSearchValueRendererTest() {
       expect(facet.enableFreezeFacetOrderFlag).toHaveBeenCalledTimes(1);
       expect(facet.scrollToTop).toHaveBeenCalledTimes(1);
       expect(dynamicFacetValue.logSelectActionToAnalytics).toHaveBeenCalledTimes(1);
+      expect(facet.logAnalyticsEvent).toHaveBeenCalledWith(analyticsActionCauseList.dynamicFacetSelect, facet.basicAnalyticsFacetMeta);
```

Comment:
This unit test is failing because it claims it's not getting called but as far as I can see it should be called through `dynamicFacetSearchValueRenderer.selectAction()` and that isn't mocked.